packages/wrangler/src/dev/dev.tsx

```diff
@@ -350,8 +424,9 @@ function DevSession(props: DevSessionProps) {
 					: undefined,
 			dev: {
 				auth: async () => {
+					const selectedAccountId = await getAccountId();
 					return {
-						accountId: await accountIdDeferred.promise,
+						accountId: selectedAccountId,
```

Comment:
With my other suggestions, we would still be able to keep the rendering of the components Local and Remote, and therefore still keep the same auth flow.

Reply:
The already of the experimentalDevEnv props passed through so can individually return null if needed